core/src/lib.rs

```diff
@@ -28,15 +28,21 @@ pub mod test_utils {
     use holochain_dna::zome::Zome;
     use holochain_dna::Dna;
     use wabt::Wat2Wasm;
-    use wabt::WabtBuf;
 
-    // Create DNA containing WASM code that returns 1337 as integer
-    pub fn create_test_dna_with_wasm() -> Dna {
-        let wasm_binary = Wat2Wasm::new()
-          .canonicalize_lebs(false)
-          .write_debug_names(true)
-          .convert(
-              r#"
+    use std::fs::File;
+
+    pub fn create_wasm_from_file(fname: &str) -> Vec<u8> {
+        use std::io::prelude::*;
+        let mut file = File::open(fname).unwrap();
+        let mut buf = Vec::new();
+        file.read_to_end(&mut buf).unwrap();
```

Comment:
`unwrap()` panics, right? i think we're trying to avoid that for recoverable errors (e.g. the file might not exist)

Reply:
Not in tests. We want tests to panic.
And anyway those unwraps are already in develop. They come from PR#44.